tests/playwright/sanity/modules/nested-accordion/nested-accordion-icon.test.ts

```diff
@@ -0,0 +1,58 @@
+import { test } from '@playwright/test';
+import WpAdminPage from '../../../pages/wp-admin-page';
+import { expectScreenshotToMatchLocator } from './helper';
+
+test.describe( 'Nested Accordion Title Icon and Text No Overlap @nested-accordion', () => {
+	test.beforeAll( async ( { browser }, testInfo ) => {
+		const context = await browser.newContext();
+		const page = await context.newPage();
+		const wpAdmin = await new WpAdminPage( page, testInfo );
+
+		await wpAdmin.setExperiments( {
+			container: 'active',
+			'nested-elements': 'active',
+		} );
+
+		await page.close();
+	} );
+
+	test.afterAll( async ( { browser }, testInfo ) => {
+		const context = await browser.newContext();
+		const page = await context.newPage();
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		await wpAdmin.setExperiments( {
+			'nested-elements': 'inactive',
+			container: 'inactive',
+		} );
+
+		await page.close();
+	} );
+
+	test( 'Nested Accordion Title Icon and Text No Overlap', async ( { browser }, testInfo ) => {
+		const page = await browser.newPage(),
+			wpAdmin = new WpAdminPage( page, testInfo ),
+			editor = await wpAdmin.openNewPage(),
+			container = await editor.addElement( { elType: 'container' }, 'document' );
+
+		let nestedAccordionID,
+			nestedAccordion;
+
+		await test.step( 'Set horizontal icon & style size to 70', async () => {
+			// Act
+			await editor.closeNavigatorIfOpen();
+			nestedAccordionID = await editor.addWidget( 'nested-accordion', container );
+			nestedAccordion = await editor.selectElement( nestedAccordionID );
+			await editor.activatePanelTab( 'content' );
+			await page.locator( '.elementor-control-icons--inline__displayed-icon' ).first().click();
+			await page.locator( '#elementor-icons-manager__search > input' ).fill( 'address card' );
```

Comment:
```suggestion
			await page.locator( '#elementor-icons-manager__search  input' ).fill( 'address card' );
```
What if someone adds a div between #elementor-icons-manager__search  and input?